### PR TITLE
fpga-projects release candidate 1.0.0

### PR DESCRIPTION
**Pull request by bmichel-psee**
Open for comments

### CODE REVIEW

**Review by lrobin-psee — APPROVED** [review]
If possible, making the same TAG for this release corresponding to the TAG in system-rtl-wip where those sources are from

**Comment by bmichel-psee** [comment]
Yes, I will do that, there is a branch un system-rtl-wip with all the sources that were used to generate this release.

**Review by cdreher-psee — APPROVED** [review]

**Review by akhamouma-psee — APPROVED** [review]

**Comment by bmichel-psee** [comment]
@lbristiel-psee we've done the technical review, can you have a look at this PR from a customer point of vue?

File: README.md [hunk 1]
```diff
@@ -1,43 +1,46 @@
...
-For additional information or support, please contact Prophesee Support at [support@prophesee.ai](mailto:support@prophesee.ai)
+## Requirements
 
-Package Structure
------------------
+The projects in this repository have been tested and validated on the following setup:
 
-### Contents
+- Ubuntu 20.04.6 LTS
+- AMD Vivado 2022.2 (64-bit) with installed support for Zynq US+ Devices
```
Comment (lbristiel-psee) [inline README.md:12]: maybe add more details on the minimal set of options that should be selected during Vivado install

**Review by lbristiel-psee — COMMENTED** [review]
I was able to generate the .xpr and then the .xsa using this PR => approved

**Review by lbristiel-psee — APPROVED** [review]

**Pull request merged**